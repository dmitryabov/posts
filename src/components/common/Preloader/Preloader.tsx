import React from 'react';
import prelouder from '../../../assets/images/spinner.svg'



let Preloader = () => {
  return <div>
       <img src={prelouder}  alt='prelouder'/>
   </div>
}

export default Preloader