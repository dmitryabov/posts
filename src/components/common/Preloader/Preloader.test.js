import React from "react";
import renderer from "react-test-renderer";
import Preloader from "./Preloader";



it(`Render Message`, () => {
  const tree = renderer
  .create(<Preloader/>, {
    createNodeMock: ()=>{
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
