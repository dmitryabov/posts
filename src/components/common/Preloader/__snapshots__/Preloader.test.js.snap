// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Message 1`] = `
<div>
  <img
    alt="prelouder"
    src="spinner.svg"
  />
</div>
`;
